doc/py_labels.rst

```diff
@@ -0,0 +1,271 @@
+.. _py_labels:
+
+Cell Labels
+===========
+
+The ``arbor.label_dict`` type is used for creating and manipulating label dictionaries,
+which can be initialised with a dictionary that defines (label, expression)
+pairs. For example, a dictionary that uses tags that correspond to SWC
+`structure identifiers <http://www.neuronland.org/NLMorphologyConverter/MorphologyFormats/SWC/Spec.html>`_
+to label soma, axon, dendrite and apical dendrites is:
+
+
+.. code-block:: python
+
+    import arbor
+
+    labels = {'soma': '(tag 1)',
+              'axon': '(tag 2)',
+              'dend': '(tag 3)',
+              'apic': '(tag 4)'}
+
+    d = arbor.label_dict(labels)
+
+Alternatively, start with an empty label dictionary and add the labels and
```

Comment:
A signal word seems appropriate to convey that both examples are the same: they are alternatives to each other.

Reply:
It's because the first part is presented as an example of a dictionary, while the second is an example of a different way of making the same dictionary; they're not really in apposition.

Reply:
I did not understand "Alternatively" to present an alternative to the example, but an alternative of the way to create the dict *as part of* the example.

I hope the following solves your confusion: "The same ``label_dict`` can be created by starting with an empty label dictionary and adding the labels and their definitions one by one:"